sbt-plugin/src/main/scala/org/scalajs/sbtplugin/ScalaJSPlugin.scala

```diff
@@ -58,6 +58,25 @@ object ScalaJSPlugin extends AutoPlugin {
     val scalaJSIsSnapshotVersion = ScalaJSVersions.currentIsSnapshot
     val scalaJSBinaryVersion = ScalaJSCrossVersion.currentBinaryVersion
 
+
+    /**
+      * Declares [[sbt.Tags.Tag Tag]]s which may be used to limit the
```

Comment:
Please follow the coding style for Scaladoc that we use. It's documented at https://github.com/scala-js/scala-js/blob/master/CODINGSTYLE.md#scaladoc